tests/test_merger.py

```diff
@@ -17,13 +17,13 @@
 sys.path.append(PROJECT_ROOT)
 
 
-def test_merge():
+def merger_operate(merger):
```

Comment:
Tests need to start with `test_`. That is how `pytest` recognizes that they are tests. Looking at your other changes, I guess you already know that? (By the way: Good work overall! Keep it up! )

```suggestion
def test_merge():
```